<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControleAgente</class>
 <widget class="QDialog" name="ControleAgente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Outros Acessos</string>
   </property>
   <widget class="QPushButton" name="btnCadastrarUsuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>126</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrar Usu치rio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnControleAcesso">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>116</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Controle Acesso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoverUsuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover Usu치rio</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Habilitar Fechadura</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>156</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxGaragem">
       <property name="text">
        <string>Porta da Garagem</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxEntrada">
       <property name="text">
        <string>Porta Entrada</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxEscritorio">
       <property name="text">
        <string>Porta Escrit칩rio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnHabilitarFechadura_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Estado Atual da Fechadura</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Porta do Escrit칩rio</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Porta da Entrada</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Porta da Garagem</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>72</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="labelEntrada">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEscritorio">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelGaragem">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAcionarAlarmeGeral">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnAcionarAlarmeGeral{
	background-color:#ff0000;
	color:#ffffff;
	border-radius:50px;
}</string>
   </property>
   <property name="text">
    <string>ACIONAR ALARME</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
